docs/using/sdk_api.md

```diff
@@ -1,10 +1,3 @@
-# `hyp3_sdk` *v1.7.3* API Reference
+# `hyp3_sdk` *v1.7.5* API Reference
 
 ::: hyp3_sdk
-
-## util
-
-::: hyp3_sdk.util
-    options:
-      show_submodules: true
-      heading_level: 3
```

Comment:
We shouldn't need this bit anymore with v1.7.4+ but I haven't verified that's true yet by building the docs and looking at the SDK API reference page

Reply:
Actually, pointing this at `develop` so we can check it on the test website; then, we can release it from there.